app/views/shared/_visn_info_modal.html.erb

```diff
@@ -0,0 +1,18 @@
+<a href="#visn-info-modal" aria-controls="visn-info-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
+  <span class="usa-sr-only">Open visn info modal</span>
+</a>
+<div class="usa-modal <%= 'display-none' if hide_modal_on_initial_load %>" id="visn-info-modal" aria-labelledby="visn-info-modal-heading" aria-describedby="visn-info-modal-description">
+  <div class="usa-modal__content">
+    <div class="usa-modal__main">
+      <div class="margin-bottom-5">
+        <p class="line-height-26 margin-top-1">
```

Comment:
I think we can use `usa-prose-body here, which will automatically assign a line height of `26px`.